packages/fxa-customs-server/package.json

```diff
@@ -16,11 +16,12 @@
     "start": "node bin/customs_server.js",
     "lint": "npm-run-all --parallel lint:*",
     "lint:deps": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
+    "lint:eslint": "eslint .",
     "test": "scripts/test-local.sh",
     "format": "prettier '**' --write"
   },
   "dependencies": {
-    "@google-cloud/pubsub": "^0.31.1",
+    "@google-cloud/pubsub": "1.1.5",
```

Comment:
Hm, a major version bump. Is this supposed to be part of this PR?

Reply:
This was an npm advisory, updating this and filed https://github.com/mozilla/fxa/issues/3376